--- a/plugins/catalog-backend-module-provider-example/src/provider/ProviderExampleProvider.ts
+++ b/plugins/catalog-backend-module-provider-example/src/provider/ProviderExampleProvider.ts
@@ -1,4 +1,5 @@
 import { Config } from '@backstage/config';
+import { serializeError, stringifyError } from '@backstage/errors';
 import {
   DeferredEntity,
   EntityProvider,
@@ -87,7 +88,7 @@
             entities,
           });
         } catch (error) {
-          logger.error(`Refresh failed`, error);
+          logger.error(`Refresh failed: ${stringifyError(error)}`, serializeError(error) as any);
         }
       },
     });
